docs/_docs/quickstart.md

```diff
@@ -75,11 +80,11 @@ You've now copied all gem files into your theme directory and removed the gem fr
 
 ## Gem-based themes and Github Pages
 
-If you're publishing your Jekyll site on [Github Pages](https://pages.github.com/), note that Github Pages supports only some gem-based themes. (Gem-based themes are a new feature.) See [Supported Themes](https://pages.github.com/themes/) in Github's documentation to see which themes are supported. (Currently only Minima is supported.) 
+If you're publishing your Jekyll site on [Github Pages](https://pages.github.com/), note that Github Pages supports only some gem-based themes. (Gem-based themes are a new feature.) See [Supported Themes](https://pages.github.com/themes/) in Github's documentation to see which themes are supported. Currently only Minima is supported.
 
 ## Using other themes
 
-The `jekyll new <sitename>` command isn't the only way to create a new Jekyll site. You can also simply [find a Jekyll theme](https://www.google.com/search?q=jekyll+themes&oq=jekyll+themes) you like, download the theme, and run the following commands to install and start Jekyll:
+The `jekyll new <sitename>` command isn't the only way to create a new Jekyll site. You can also simply find a [Jekyll theme](../themes/) you like, download the theme, and run the following commands to install and start Jekyll:
```

Comment:
=> `jekyll new <PATH>`
=> `(../themes/)` is not a source to find more themes.. its a `documentation page for what Jekyll Themes are all about`.. which is why I suggested you move this section and related sections on *Gem-based Themes* to that page.. and then link to that page here.. 
=> Browse for more themes at `https://github.com/jekyll/jekyll/wiki/Themes`